specifyweb/frontend/js_src/lib/components/WbPlanView/mappingPreview.ts

```diff
@@ -176,5 +177,5 @@ export function generateMappingPathPreview(
     toManyIndexFormatted,
   ])
     .filter(Boolean)
-    .join(' · ');
+    .join(addSeparator ? ' · ' : ' ');
```

Comment:
instead of adding this pref should we just pick a different separator?
like a `>` or `-` or some other ascii character that avoids this issue entirely?
that would avoid a needless pref

so either that, or make the pref into a string to allow fuller customization.